src/validation.cpp

```diff
@@ -2616,7 +2616,7 @@ bool CChainState::ConnectBlock(const CBlock& block, CValidationState& state, CBl
                     // Cast to avoid UniValue in GovVariable Export/Import
                     auto lpVar = dynamic_cast<LP_DAILY_DFI_REWARD*>(var.get());
                     if (lpVar && lpVar->dailyReward != subsidy) {
-                        lpVar->dailyReward = subsidy;
+                        lpVar->dailyReward = subsidy * ((60 * 60 * 24) / chainparams.GetConsensus().pos.nTargetSpacing);
```

Comment:
We have `BlocksPerDay` in Consensus.
https://github.com/DeFiCh/ain/blob/master/src/consensus/params.h#L131-L134

Reply:
Updated to use BlocksPerDay.